### PR TITLE
Anthony patch1

### PR DESCRIPTION
**Pull request by AnthonyAmanse**
Added secrets for use in Spring Boot Microservices. 
Updated `wsk api-experimental` to `wsk api`

### CODE REVIEW

**Comment by animeshsingh** [comment]
@jzaccone @Tomcli Please review it.

File: README.md [hunk 1]
```diff
@@ -45,49 +45,97 @@ The backend consists of the MySQL database and the Spring Boot app. You will als
 * There are two ways to create the MySQL database backend: **Use MySQL in a container in your cluster** *OR* **Use Bluemix MySQL**
 
 ## 1.1 Use MySQL in container
-**NOTE:** Leave the environment variables blank in the `compute-interest-api.yaml` and `account-summary.yaml`
 ```bash
 $ kubectl create -f account-database.yaml
 service "account-database" created
 deployment "account-database" created
 ```
+Default credentials are already encoded in base64 in secrets.yaml.
```
Comment (jzaccone) [inline README.md]: We should probably put a disclaimer in there about best practices when managing sensitive information. 1) Don't store in github 2) base64 does not hide your secrets 3) Use password manager and kube secrets which separate your coding flow from your secrets management

**Comment by jzaccone** [comment]
Other than the one comment, LGTM.

**Comment by animeshsingh** [comment]
After @Tomcli has time to review it, and @AnthonyAmanse adds your suggested disclaimer, please go ahead and merge it @jzaccone

File: README.md [hunk 2]
```diff
@@ -228,11 +278,11 @@ env:
 - name: EMAIL_RECEIVER
   value: 'sendTo@gmail.com' # the receiver's email
 - name: OPENWHISK_API_URL_SLACK
-  value: 'https://XXX-YYY-ZZZ-gws.api-gw.mybluemix.net/v1/slack' # your API endpoint for slack notifications
+  value: 'https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/.../v1/slack' # your API endpoint for slack notifications
 - name: SLACK_MESSAGE
   value: 'Your balance is over $50,000.00' # your custom message
 - name: OPENWHISK_API_URL_EMAIL
-  value: 'https://XXX-YYY-ZZZ-gws.api-gw.mybluemix.net/v1/email' # your API endpoint for email notifications
+  value: 'hhttps://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/.../v1/email' # your API endpoint for email notifications
```
Comment (Tomcli) [inline README.md]: also extra h in https

**Pull request merged**